lib/timing.js

```diff
@@ -20,15 +24,16 @@ module.exports = function(event, checkTime = true) {
         const currentTime = moment.tz('Europe/Berlin');
 
         const currentDay = currentTime.isoWeekday(); // 1 = Monday, 7 = Sunday
+        const isHoliday = currentTime.isHoliday('NRW');
 
         let expectedTime = null;
 
-        if (timing === 'morning' && 1 <= currentDay && currentDay <= 5) {
-            expectedTime = moment(currentTime).hour(7).minute(30);
-        } else if (timing === 'morning' && 6 <= currentDay && currentDay <= 7) {
-            expectedTime = moment(currentTime).hour(9).minute(0);
+        if (timing === 'morning' && (isHoliday || 6 <= currentDay && currentDay <= 7)) {
```

Comment:
This condition is really hard to understand and might lead to weird behavior in edge cases.
Consider breaking out the weekend day check into a variable to make understandable.
Also: It might make sense to use an outer if clause for checking the timing variable an then checking further parameters inside of the `timing === 'morning'` branch